confuse.py

```diff
@@ -566,13 +724,17 @@ def __init__(self, sources):
         self.redactions = set()
 
     def add(self, obj):
-        self.sources.append(ConfigSource.of(obj))
+        src = ConfigSource.of(obj)
+        self.sources.append(src)
+        return src
 
     def set(self, value):
-        self.sources.insert(0, ConfigSource.of(value))
+        src = ConfigSource.of(value)
+        self.sources.insert(0, src)
+        return src
 
     def resolve(self):
-        return ((dict(s), s) for s in self.sources)
+        return ((dict(s.load()), s) for s in self.sources)
```

Comment:
Perhaps `dict(s)` on a source should implicitly call `load`? (In fact, maybe it already does?)

Reply:
Yeah it totally does - and all tests work fine locally, except in tox/travis python 2.7 tests... 

But the weird thing is, they work fine with pytest and nosetests with python 2....

So maybe there's some weird environment thing ???? idk I'm kinda stumped.

I only added it for that one test, but if we can show that it's some environment fluke or something, we can omit it.